### PR TITLE
Feat/yarn

### PR DESCRIPTION
**Pull request by jordan-a-young**
I added the lib [standard-version](https://github.com/conventional-changelog/standard-version) to help with easier releases.

`standard-version` uses conventional commits to determine version bump and generate a changelog.

I also turned off `unicorn/prefer-query-selector`.

I ran `yarn release` and pushed the tags, but have not run `yarn publish` yet.

### CODE REVIEW

**Comment by GoPro16** [comment]
You can delete the `release` script now since we are removing that.

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,6094 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.5.5":
+  version "7.5.5"
+  resolved "https://artifactory.availity.com/artifactory/api/npm/npm/@babel/code-frame/-/code-frame-7.5.5.tgz#bc0782f6d69f7b7d49531219699b988f669a8f9d"
```
Comment (GoPro16) [inline yarn.lock]: Should not be pointing to our proxy actually.

Comment (jordan-a-young) [reply yarn.lock]: I need to run unproxy then install again.

**Comment by GoPro16** [comment]
You can merge it whenever you get back on and then run the publish command.

**Pull request merged**